vm-image-spec.yaml

```diff
@@ -16,7 +16,11 @@ commands:
   - name: sql-exporter
     user: nobody
     sysvInitAction: respawn
-    shell: '/bin/sql_exporter -config.file=/etc/sql_exporter.yml'
+    shell: '/bin/sql_exporter -config.file=/etc/sql_exporter.yml -web.listen-address=:9399'
```

Comment:
changing this line looks sketchy to me (is it the same port before and after?) but everything else lgtm

(if it's worth being explicit about the listen address for the "normal" sql-exporter, may be better to do that in a separate change. idk - ultimately that part is probably owned by compute, so if you feel this is right, go for it)

Reply:
It's the same port and i'd like to make it explicit. I can separate another PR.

Reply:
Your call — either way, I'd test on staging that the metrics still show up in the console (or are available via the "per-VM metrics" dashboard in grafana)

Reply:
Yep, I’ll merge after next week’s platform release to have enough time testing it in staging.